_config.yml

```
@@ -17,6 +17,9 @@ exclude:
 
 twitter_username: opensrcdesign
 github_username:  opensourcedesign
+mastodon:
+- mastodon_username: @opensourcedesign
+  instance: mastodon.social
```

This is what's needed:
```suggestion
mastodon_username: @opensourcedesign
```